cost/scheduled_reports_with_estimates/README.md

```diff
@@ -0,0 +1,57 @@
+# Scheduled Report with Estimates
+
+## What it does
+
+This policy allows you to set up scheduled reports that will provide summaries of cloud cost across all resources in the billing centers you specify, delivered to any email addresses you specify. The policy will report the following:
+
+Chart of the previous 6 months of utilization based on whichever Reporting Dimension you select (only bill data and RightScale-generated dimensions are supported).
+Daily Average - Weekly: Daily average costs calculated from Monday of the previous week through today. Daily Average - Monthly: Daily average costs calculated from the 1st of the previous month through today. Previous - Weekly: Total costs during previous full week (Monday-Sunday).
+Previous - Monthly: Total costs during previous full month.
+Current - Weekly: Total costs during current (incomplete) week.
+Current - Monthly: Total costs during current (incomplete) month.
+Estimated Cost for each metrics for the next 3 months based on previous costs.
+
+We recommend running this policy on a weekly cadence and applying it to your master account.
+
+Note 1: The last 3 days of data in the current week or month will contain incomplete data.
+Note 2: The account you apply the policy to is unimportant as Optima metrics are scoped to the Org.
+
+Cost Metrics
```

Comment:
needs heading

Reply:
To add a heading you need to add ## in front of Cost Metrics

Reply:
done